tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Input/ObjcInterop/ObjcInteropTypeReader.swift

```diff
@@ -0,0 +1,197 @@
+/*
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+* This product includes software developed at Datadog (https://www.datadoghq.com/).
+* Copyright 2019-2020 Datadog, Inc.
+*/
+
+import Foundation
+
+/// Reads `ObjcInteropType` definitions from `SwiftType` definitions.
+internal class ObjcInteropTypeReader {
+    /// `SwiftTypes` passed on input.
+    private var inputSwiftTypes: [SwiftType] = []
+    /// `ObjcInteropTypes` returned on output.
+    private var outputObjcInteropTypes: [ObjcInteropType] = []
+
+    func readObjcInteropTypes(from swiftTypes: [SwiftType]) throws -> [ObjcInteropType] {
```

Comment:
Similar to my comment above, that sounds a bit confusing to me. Maybe not ideal either but how about the following:
```suggestion
    func objcInteropTypes(from swiftTypes: [SwiftType]) throws -> [ObjcInteropType] {
```

Reply:
Agree 👍, this is not "reading" per se. As this violation is not local to only `ObjcInteropTypeReader` (it's there also in `SwiftTypeReader`), let me do wider renaming / refactor in upcoming PR on this topic 🙏.

Reply:
Done in Done in https://github.com/DataDog/dd-sdk-ios/pull/386